### DISCUSSION TITLE
Close nav on link click

### DISCUSSION THREAD

**Discussion by CHolitas** [swup/swup#390] [Help]
Hello! I wanted to use Swipe in my website, the thing is that the Naw doesn't closes with can confuse the user on mobile, can somebody help me to close the nav-bar

**Comment by AugustMiller** [comment]
If you have a function that opens or closes you navigation element, you could put a call to it inside a Swup event listener:

```js
const swup = new Swup(...);

swup.on('clickLink', (e) => {
  myCloseNavFunction();
});
```

Keep in mind that this will have to be attached in a scope where your nav open/close functions are defined (or your logic will have to be duplicated into the handler, i.e. `$('.navigation').hide()`).

Take a look at the [available events](https://swup.js.org/events), if you'd like to trigger the close behavior at a more specific step (i.e. when animation begins).

**Reply by CHolitas** [reply]
But I have label for nav-toggle

**Reply by AugustMiller** [reply]
Could you elaborate on that?

If you only have a function that toggles your nav open and closed, you'll need to split it into two, such that you can request it be "closed" from whatever state it's in, whenever a navigation occurs.
